src/agoradatatools/etl/utils.py

```diff
@@ -142,6 +157,20 @@ def nest_fields(
         .rename(columns={0: new_column})
     )
 
+    if nested_field_is_list:
+        return nested
+
+    # nested_field_is_list == False
+    lengths = nested[new_column].apply(len)
+    if all(lengths == 1):
```

Comment:
possible future proofing - if we someday had a scenario where there could be either 1 dictionary or 0 dictionaries, would simply changing `lengths == 1` to `lengths <= 1` here handle that?

Reply:
Could you give me an example where you'd get 0 dictionaries? I think if you use `pd.group_by`, by definition if a group exists it has to be in at least one row?

Reply:
I cannot, I was thinking abstractly...If `group_by` never returns groupings of 0, then we don't need to worry about handling that case and you can ignore my feedback about handling it entirely :)